src/services/doppler-legacy-client.ts

```diff
@@ -16,7 +16,10 @@ export interface DopplerLegacyClient {
 type UnexpectedError = { success?: false; message?: string | null; error?: any };
 type ErrorResult<TError> = { success?: false; expectedError: TError } | UnexpectedError;
 type Result<TResult, TError> = { success: true; value: TResult } | ErrorResult<TError>;
-type EmptyResult<TError> = { success: true } | ErrorResult<TError>;
+type EmptyResult<TError> =
+  | { success: true }
+  | { success: true; redirectUrl?: string }
```

Comment:
_EmptyResult_ tries to refer to a generic result without output payload. I think that it is not a good idea change it in this way.